.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -259,7 +268,36 @@ main() {
     exit 1
   fi
 
-  run_tests
+  run_tests "${NAME_SPACE}"
+}
+
+main_rbac() {
+  configure_namespace ${NAME_SPACE_RBAC}
+  installPipelinesOperator "${DIR}"
+  install_helm
+  uninstall_helmchart
+
+  cd "${DIR}"
+  apply_yaml_files "${DIR}" ${NAME_SPACE_RBAC}
+  add_helm_repos
+
+  echo "Tag name : ${TAG_NAME}"
+
+  helm upgrade -i ${NAME_SPACE_RBAC} -n ${NAME_SPACE_RBAC} rhdh-chart/backstage --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=${TAG_NAME}
```

Comment:
```suggestion
  helm upgrade -i ${RELEASE_NAME} -n ${NAME_SPACE_RBAC} ${HELM_REPO_NAME}/${HELM_IMAGE_NAME} --version ${CHART_VERSION} -f $DIR/value_files/${HELM_CHART_VALUE_FILE_NAME} --set global.clusterRouterBase=${K8S_CLUSTER_ROUTER_BASE} --set upstream.backstage.image.tag=${TAG_NAME}
```

Please update the Helm upgrade as suggested. The current command appears incorrect; we need to include the release name and use the predefined environments for rhdh-chart/backstage as suggested above.

Reply:
@subhashkhileri release name can be anything, we can just choose any name that we want

Reply:
@subhashkhileri updated